VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegment.java

```diff
@@ -8,6 +8,8 @@
 
 import org.edx.mobile.util.images.ShareUtils;
 
+import java.util.Map;
+
 public interface ISegment {
```

Comment:
Out of scope for this ticket, but at some point we should rename this to something more generic like ``Analytics`` Using segment is an implementation detail